lib/Model/index.ts

```diff
@@ -239,6 +239,7 @@ interface ModelBatchPutSettings {
 interface ModelUpdateSettings {
 	return?: "document" | "request";
 	condition?: Condition;
+	returnValues?: DynamoDB.ReturnValue;
```

Comment:
Super clever. Part of me really loves this. Less to maintain, and passing that responsibility onto DynamoDB.

My only feeling is that this type of concept of exposing DynamoDB types within a Dynamoose type isn't done at all before this. So I'm slightly hesitant to do this. But I can't think of any good argument for why it could be a bad thing. If AWS were to break it somehow, we have much bigger issues.